packages/datadog-plugin-jest/src/index.js

```diff
@@ -60,6 +79,11 @@ function createHandleTestEvent (tracer, testEnvironmentMetadata) {
       [SAMPLING_PRIORITY]: AUTO_KEEP,
       ...testEnvironmentMetadata
     }
+    let testParameters = params[event.test.name]
+    if (testParameters) {
+      testParameters = testParameters.shift()
+      commonSpanTags[TEST_PARAMETERS] = JSON.stringify(testParameters)
```

Comment:
Why use a single string tag here? Splitting in multiple tags would allow an object-like view in the UI.